impeller/renderer/backend/vulkan/command_pool_vk.cc

```diff
@@ -3,166 +3,195 @@
 // found in the LICENSE file.
 
 #include "impeller/renderer/backend/vulkan/command_pool_vk.h"
+#include <memory>
+#include <optional>
+#include <utility>
+#include "fml/macros.h"
+#include "fml/thread_local.h"
+#include "fml/trace_event.h"
+#include "impeller/renderer/backend/vulkan/resource_manager_vk.h"
+#include "vulkan/vulkan_structs.hpp"
```

Comment:
Like commented elsewhere, include the `vk.h` umbrella headers. For Vulkan, VMA, OpenGL, EGL, etc..

Reply:
Done.